crates/ide_assists/src/handlers/unmerge_use.rs

```diff
@@ -1,3 +1,5 @@
+use ast::make;
```

Comment:
auto-import haven’t figured out that we use ast::self already?

Reply:
Huh that is odd. I think I used the qualify path assist here, so maybe tha has still problems with this.